IFC4x3/Properties/s/SurfaceType_0tYpLvR3jCvOyVlWsiP1tv/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SurfaceType_0tYpLvR3jCvOyVlWsiP1tv" Name="SurfaceType" UniqueId="378b3579-6c3b-4ce5-8f1f-be0dac641df9" PrimaryDataType="IfcLabel" />
```

Comment:
Propose using the newly added type objects to model this.

Reply:
Yes, I agree!

Reply:
Removed SurfaceType.